packages/bundle-size/package.json

```diff
@@ -5,12 +5,16 @@
     "report": "tsx src/report.ts",
     "pregenerate": "rm -rf src/gen",
     "generate": "buf generate buf.build/googleapis/googleapis:9475e2896f8a46d09806149f9382e538",
-    "postgenerate": "license-header ."
+    "postgenerate": "license-header .",
+    "format": "prettier --write --ignore-unknown '.' '!src/gen/**' '!.turbo'",
+    "license-header": "license-header",
+    "lint": "eslint --max-warnings 0 ."
```

Comment:
Tools like prettier are better run for each package individually with turborepo so that it can cache properly.